### PR TITLE
Added new functions to run oracle tests and made changes in env.sh and init-db of the tests

### PR DESCRIPTION
**Pull request by ShivanshGahlot**

### CODE REVIEW

File: migtests/scripts/functions.sh [hunk 1]
```diff
@@ -132,7 +160,23 @@ run_sqlplus() {
...
+    conn_string="sys/password@${SOURCE_DB_HOST}:${SOURCE_DB_PORT}/${db_name} as SYSDBA"
+	echo exit | sqlplus -f "${conn_string}" @"${file_name}"
+}
+
+
+run_sqlplus_write() {
+    db_name=$1
+    sql=$2
+    conn_string="test_user/password@${SOURCE_DB_HOST}:${SOURCE_DB_PORT}/${db_name}"
+    echo exit | sqlplus -f "${conn_string}" @"${sql}"
```
Comment (amit-yb) [inline migtests/scripts/functions.sh:179]: Just the way we did with PG, let's have two functions: run_sqlplus() and run_sqlplus_admin().
The first should connect with $SOURCE_DB_USER and $SOURCE_DB_PASSWORD. And the later should connect using $SOURCE_DB_ADMIN_USER and $SOURCE_DB_ADMIN_PASSWORD.

File: migtests/scripts/functions.sh [hunk 2]
```diff
@@ -132,7 +160,23 @@ run_sqlplus() {
...
+
+run_sqlplus_grants() {
+	db_name=$1
+    local file_name=$2
+    conn_string="sys/password@${SOURCE_DB_HOST}:${SOURCE_DB_PORT}/${db_name} as SYSDBA"
+	echo exit | sqlplus -f "${conn_string}" @"${file_name}"
+}
+
+
+run_sqlplus_write() {
```
Comment (amit-yb) [inline migtests/scripts/functions.sh:175]: How is this function different from the `run_sqlplus()` on the line 159?

File: migtests/scripts/functions.sh [hunk 3]
```diff
@@ -106,11 +106,39 @@ grant_user_permission_mysql() {
...
+        		EXECUTE IMMEDIATE 'grant select on '||R.owner||'.\"'||R.object_name||'\" to ybvoyager';
+  			END LOOP;
+		END;
+		/"
+	)
+
+	file_name="oracle-inputs.sql"
+	for command in "${commands[@]}"; do
+		echo "${command}" >> "${file_name}"
+	done
```
Comment (amit-yb) [inline migtests/scripts/functions.sh:133]: Why do we need an array and a loop for this? Can't we use simple shell redirection?
```
cat > oracle-inputs.sql << EOF
grant connect to $SOURCE_DB_USER
grant ...
...
BEGIN
  ...
END
/
BEGIN
  ...
END
/

EOF
```

File: migtests/tests/oracle/bi-db/env.sh [hunk 4]
```diff
@@ -1,3 +1,7 @@
 export SOURCE_DB_TYPE="oracle"
 export SOURCE_DB_SCHEMA=${SOURCE_DB_SCHEMA:-"BI"}
 export TARGET_DB_NAME=${TARGET_DB_NAME:-"bi_db_test"}
+export SOURCE_DB_USER_SCHEMA_OWNER=${SOURCE_DB_USER_SCHEMA_OWNER:-"TEST_SCHEMA"}
+export SOURCE_DB_USER_SCHEMA_OWNER_PASSWORD=${SOURCE_DB_USER_SCHEMA_OWNER_PASSWORD:-"password"}
+export SOURCE_DB_USER_SYS=${SOURCE_DB_USER_SYS:-"SYS"}
```
Comment (priyanshi-yb) [inline migtests/tests/oracle/bi-db/env.sh:6]: Add these to the main `oracle/env.sh` and remove them from test-specific `env.sh`

Comment (ShivanshGahlot) [reply migtests/tests/oracle/bi-db/env.sh]: Done

File: migtests/scripts/functions.sh (hunk 3 above)
Comment (ShivanshGahlot) [reply migtests/scripts/functions.sh]: Done

**Pull request merged**